packages/usehooks-ts/src/useTernaryDarkMode/useTernaryDarkMode.ts

```diff
@@ -26,36 +27,14 @@ interface UseTernaryDarkModeOutput {
  */
 export function useTernaryDarkMode(
   localStorageKey: string = LOCAL_STORAGE_KEY,
+  defaultValue: TernaryDarkMode = 'system',
```

Comment:
I've resolve the coflicts here and put it as the second param to avoid introducing a breaking change.

However, this will also mean that to set a default value, people will have to specify a `localStorageKey`, even if they just want to use the default key.

I would be in favour of introducing object-params here so that it's possible to specify whichever optional params you want, like

```ts
useTernaryDarkMode({ defaultValue: 'light' });
```

which avoids having to re-specify the default local storage key just to set a default

```ts
useTernaryDarkMode(LOCAL_STORAGE_KEY, 'light')
```

We could do this without it being a breaking change with method overloading. What do you think @juliencrn?

Reply:
Hi @RobHannay,
Thanks for the PR (and its recent update). 

Adding a `defaultValue` and making an object params is a great idea 👍 
It effectively introduces API change, so I've indeed used function overload to accept both the old and the new interface. The old one will be dropped in the v3